src/components/ReviewsByCategory.jsx

```diff
@@ -0,0 +1,35 @@
+import Reviews from "./Reviews";
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchReviews } from "../utils/api";
+
+export function ReviewsByCategory() {
+  const { category } = useParams();
+  const [reviewsByCategory, setReviewsByCategory] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchReviews().then((fetchedReviews) => {
+      const reviewsOfCategory = [];
+      fetchedReviews.forEach((fetchedReview) => {
+        if (fetchedReview.category === category) {
+          reviewsOfCategory.push(fetchedReview);
+        }
```

Comment:
we shouldn't be using our frontend to filter through the reviews - we have an endpoint on our backend to deal with queries which we should implement here in your fetchReviews function